runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process/process.json

```diff
@@ -328,6 +328,9 @@
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.FilePath.Info": "Refers to the location path, where the starting point is the folder, where the bot lands after login to the account, e.g. folderName/fileName.xlsx",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.ListName.Info": "It must be a resource of type 'Document library'",
     "Process.Details.Modeler.Actions.CloudExcel.CloseSession.Label": "Close session",
+    "Process.Details.Modeler.Actions.CloudExcel.CreateWorksheet.Label": "Create worksheet",
+    "Process.Details.Modeler.Actions.CloudExcel.CreateWorksheet.WorksheetName": "Worksheet name",
+    "Process.Details.Modeler.Actions.CloudExcel.CreateWorksheet.WorksheetName.Info": "Maximum number of characters: 31. Characters not allowed: \\ , / , * , ? , : , [ , ]",
```

Comment:
Imo +"New worksheet name. Maximum 31 characters. Characters not allowed: \\ , / , * , ? , : , [ , ]",

Reply:
but it's just a suggestion